src/js/providers/utils/quality-labels.ts

```diff
@@ -11,6 +11,7 @@ export function generateLabel(level?: ProviderLevel, qualityLabels?: QualityLabe
         return '';
     }
     // flash provider uses bitrate instead of bandwidth
+    // Because these may be used by custom providers, they were not removed as part of flash deprecation
```

Comment:
Just change the first comment
```suggestion
    // custom providers may use level bitrate rather than bandwidth
```